### DISCUSSION TITLE
AX6编译5.10-mac80211错误

### DISCUSSION THREAD

**Discussion by jliveusa** [Boos4721/openwrt#96] [Bugs/问题反馈]
5.10内核下，mac80211依赖kmod-qca-nss-drv 、kmod-qca-nss-drv-64，两者同时选定会出现安装冲突错误，删掉kmod-qca-nss-drv-64后编译成功，但刷机后无法连接WAN口，wifi无法启动。

**Comment by Boos4721** [comment]
yup, any logs?
